### PR TITLE
AAP 2.4 Translations (2) 

### PR DESCRIPTION
**Pull request by rooftopcellist**
#### What is this PR doing:

Localization updates for AAP 2.4.
* pulled - 2023 05 09 18 15 20

Replaces https://github.com/ansible/galaxy_ng/pull/1715

#### Reviewers must know:

**PR Author & Reviewers**: Keep or remove backport labels per [Backporting Guidelines](https://github.com/ansible/galaxy_ng/wiki/Backporting-Guidelines)
**Reviewers**: Look for sound code, no [code smells](https://www.codegrip.tech/productivity/everything-you-need-to-know-about-code-smells/), docs & test coverage
**Merger**: When merging, include the Jira issue link in the squashed commit

### CODE REVIEW

**Review by jerabekjiri — APPROVED** [review]

File: galaxy_ng/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -396,476 +356,512 @@ msgstr "Un espace de nom nommé %s existe déjà."
...
+#: app/constants.py:257
 msgid "Delete task"
-msgstr ""
+msgstr "Supprimer la tâche"
 
-#: app/constants.py:238
+#: app/constants.py:258
 msgid "Delete this task."
-msgstr ""
+msgstr "Supprimer ce nœud."
```
Comment (himdel) [inline galaxy_ng/locale/fr/LC_MESSAGES/django.po:848]:
```suggestion
msgstr "Supprimer ce tâche."
```

?

File: galaxy_ng/locale/es/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -392,476 +356,512 @@ msgstr "Ya existe un espacio de nombres llamado %s."
...
 
 #: app/constants.py:75
 msgid "Change user"
-msgstr ""
+msgstr "Cambiar usuario"
 
 #: app/constants.py:76
 msgid "Edit this user."
-msgstr ""
+msgstr "Modificar este nodo"
```
Comment (himdel) [inline galaxy_ng/locale/es/LC_MESSAGES/django.po:469]:
```suggestion
msgstr "Modificar este usuario"
```

File: galaxy_ng/locale/es/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -392,476 +356,512 @@ msgstr "Ya existe un espacio de nombres llamado %s."
...
 
 #: app/constants.py:81
 msgid "Delete user"
-msgstr ""
+msgstr "Eliminar usuario"
 
 #: app/constants.py:82
 msgid "Delete this user."
-msgstr ""
+msgstr "Eliminar este nodo"
```
Comment (himdel) [inline galaxy_ng/locale/es/LC_MESSAGES/django.po:481]:
```suggestion
msgstr "Eliminar este usuario"
```

File: galaxy_ng/locale/es/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -392,476 +356,512 @@ msgstr "Ya existe un espacio de nombres llamado %s."
...
 
 #: app/constants.py:99
 msgid "Change group"
-msgstr ""
+msgstr "Cambiar de grupo"
 
 #: app/constants.py:100
 msgid "Edit this group"
-msgstr ""
+msgstr "Modificar este nodo"
```
Comment (himdel) [inline galaxy_ng/locale/es/LC_MESSAGES/django.po:518]:
```suggestion
msgstr "Modificar este grupo"
```

File: galaxy_ng/locale/es/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -392,476 +356,512 @@ msgstr "Ya existe un espacio de nombres llamado %s."
...
 
 #: app/constants.py:105
 msgid "Delete group"
-msgstr ""
+msgstr "Eliminar grupo"
 
 #: app/constants.py:106
 msgid "Delete this group."
-msgstr ""
+msgstr "Eliminar este nodo"
```
Comment (himdel) [inline galaxy_ng/locale/es/LC_MESSAGES/django.po:530]:
```suggestion
msgstr "Eliminar este grupo"
```

**Comment by himdel** [comment]
There seems to be some inconsistency with the translation of "Remotes" .. mando, vs remoto, vs "a distancia", is that something worth fixing?

**Comment by newswangerd** [comment]
@rooftopcellist can you update the PR with @himdel's suggestions?

File: galaxy_ng/locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (dsavineau) [reply galaxy_ng/locale/fr/LC_MESSAGES/django.po:848]: probably more:

Comment (rooftopcellist) [reply galaxy_ng/locale/fr/LC_MESSAGES/django.po:848]: The perks of having a French speaker on the team! :heart:

**Comment by rooftopcellist** [comment]
@himdel I updated the PR with those suggestions, and a couple of the "remote" discrepancies that I was able to understand enough to change.  

it looks like CI failed on something unrelated

**Pull request merged**